extensions/meme_rater.py

```diff
@@ -66,6 +63,15 @@ async def main(event: hikari.GuildMessageCreateEvent) -> None:
             ratings.append(int(rating))
         except Exception as e:
             continue
+    
+    for match in IMAGE_URL_REGEX.finditer(event.message.content):
+        image_url = match.group(1)
+        try:
+            rating = await get_meme_rating(image_url, event.author.username, att_ext)
```

Comment:
Should I be vaguely concerned that the bot is now talking to arbitrary servers?

Reply:
I can look into getting a url that has been cached by discord's media proxy